components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -239,6 +240,7 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
                             aria-label={Utils.localizeMessage('post_info.submenu.icon', 'submenu icon').toLowerCase()}
                         />}
                     {subMenuContent}
+                    {extraText && <span className='MenuItem__help-text'>{extraText}</span>}
```

Comment:
`helpText` would actually make way more sense. and you already have that as a class-name, sooooo ... :D

Reply:
And it would probably be better to structure it like this:

```typescript
<div
    className={classNames([{styleSelectableItemDiv: styleSelectableItem}])}
    id={id}
    aria-label={ariaLabel}
    onMouseEnter={this.show}
    onMouseLeave={this.hide}
    onClick={this.onClick}
    tabIndex={0}
    onKeyDown={this.handleKeyDown}
>
    {extraText ? <div>{textprop}<span className='MenuItem__help-text'>{extraText}</span></div> : textProp}
    {selectedValueText && <span className='selected'>{selectedValueText}</span>}
    {id !== 'SidebarChannelMenu-moveToDivider' &&
        <span
            id={'channelHeaderDropdownIconRight_' + id}
            className={classNames([`fa fa-angle-right SubMenu__icon-right${hasSubmenu ? '' : '-empty'}`, {mobile: isMobile}])}
            aria-label={Utils.localizeMessage('post_info.submenu.icon', 'submenu icon').toLowerCase()}
        />}
    {subMenuContent}
</div>
```

But have the `div` a column character.